lisa/sut_orchestrator/azure/common.py

```diff
@@ -2137,70 +2121,61 @@ def check_certificate_existence(
     vault_url: str,
     cert_name: str,
     log: Logger,
-) -> bool:
-    certificate_client = get_certificate_client(vault_url, platform)
-
-    try:
-        certificate_client.begin_delete_certificate(cert_name)
-        log.info(f"Certificate {cert_name} deleted successfully.")
-        return True
-    except Exception as e:
-        log.error(f"Error deleting certificate {cert_name}: {e}")
-        return False
-
-
-def rotate_certificates(
-    log: Logger,
-    vault_url: str,
-    platform: "AzurePlatform",
-    cert_name_to_rotate: str,
 ) -> None:
-    # Use the helper function to get certificate_client
     certificate_client = get_certificate_client(vault_url, platform)
 
     # Retrieve the old version of the certificate
-    if not certificate_client.get_certificate(cert_name_to_rotate):
-        error_message = (
-            f"Failed to retrieve old version of certificate: {cert_name_to_rotate}"
-        )
-        log.error(error_message)
+    if not certificate_client.get_certificate(cert_name):
+        error_message = f"Failed to retrieve old version of certificate: {cert_name}"
         raise LisaException(error_message)
 
     cert_policy = CertificatePolicy.get_default()
 
     # Create only the specified certificate
-    for _ in range(2):
-        # Create certificate
-        create_certificate_poller = certificate_client.begin_create_certificate(
-            cert_name_to_rotate, policy=cert_policy
+    # Create certificate
+    create_certificate_poller = certificate_client.begin_create_certificate(
+        cert_name, policy=cert_policy
+    )
+    create_certificate_result = create_certificate_poller.result()
+
+    # Handle possible None value
+    if (
+        isinstance(create_certificate_result, KeyVaultCertificate)
+        and hasattr(create_certificate_result, "properties")
+        and create_certificate_result.properties
+    ):
+        new_certificate_version = create_certificate_result.properties.version
+        log.debug(
+            f"New version of certificate '{cert_name}': " f"{new_certificate_version}"
         )
-        create_certificate_result = create_certificate_poller.result()
-
-        # Handle possible None value
-        if (
-            isinstance(create_certificate_result, KeyVaultCertificate)
-            and hasattr(create_certificate_result, "properties")
-            and create_certificate_result.properties
-        ):
-            new_certificate_version = create_certificate_result.properties.version
-            log.info(
-                f"New version of certificate '{cert_name_to_rotate}': "
-                f"{new_certificate_version}"
-            )
-            log.info("Certificate rotated")
-        else:
-            error_message = (
-                "Failed to retrieve properties from create certificate result."
-            )
-            log.error(error_message)
-            raise LisaException(error_message)
+        log.debug("Certificate rotated")
```

Comment:
Merge this line with above one. Avoid printing two lines of logs together. Each line of log should place on different code paths, unless there are too much information.